core-bundle/src/Cron/Cron.php

```diff
@@ -127,7 +127,7 @@ private function doRun(array $cronJobs, string $scope, bool $force = false): voi
         $entityManager = ($this->entityManager)();
         $cronJobsToBeRun = [];
 
-        $now = new \DateTimeImmutable();
+        $now = new \DateTime();
```

Comment:
Why this change?🤔

Reply:
See the [initial description](https://github.com/contao/contao/pull/6859#issue-2119617501):

> The changes for the `CronJob` entity are necessary due to the stricter `datetime` types in `doctrine/dbal: 4.x`. We used `\DateTimeImmutable` on a `datetime` field erroneously. This PR fixes that in a BC way.